packages/manager/modules/pci/src/projects/project/data-processing/submit-job/translations/Messages_fr_FR.json

```diff
@@ -10,5 +10,22 @@
   "data_processing_submit_job_stepper_submit_error_title": "Une erreur est survenue",
   "data_processing_submit_job_button_label": "Démarrer le job",
   "data_processing_submit_job_spinner_label": "lancement en cours…",
-  "data_processing_submit_job_error_label": "Une erreur est survenue, nous ne pouvons pas démarrer votre job pour le moment."
+  "data_processing_submit_job_error_label": "Une erreur est survenue, nous ne pouvons pas démarrer votre job pour le moment.",
+  "Client::UnprocessableEntity::odp.job.engine.unsupportedValue": "Erreur de saisie : le type de job demandé n'est pas supporté",
```

Comment:
Maybe we can stick with a translation key written in snakeCase.

Reply:
Commit amended with correction